### PR TITLE
I think we should add a file for Ocala, FL

### PR DESCRIPTION
**Pull request by githubteacher**
This file needs to go through the workflow:
- [x] Create branch
- [x] Add some commits
- [x] Open a pull request
- [x] Have a discussion
- [ ] Merge my pull request

### CODE REVIEW

**Comment by githubteacher** [comment]
@Bedbath can you please put some :eyes: on this for me? Please?

File: uniquedeep-balaju.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Hello All, Kathmandu Nepal is one of the best place to live in. 
+
+- High Himalayas
```
Comment (githubteacher) [inline uniquedeep-balaju.md:3]: Do you enjoy mountain climbing?

Comment (UniqueDeep) [inline uniquedeep-balaju.md:3]: Of course!!!

Comment (githubteacher) [inline uniquedeep-balaju.md:3]: We need photos!

**Pull request merged**